### PR TITLE
fix: jira changelog.FromValue & changelog.ToValue

### LINKED ISSUE #2691: [Bug][Jira] Incorrect values of `issue_changelogs.from_value` and `issue_changelogs.to_value`
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar issues.

### What happened
the columns `issue_changelogs.from_value` and `issue_changelogs.to_value` were assigned wrong value for the data with `issue_changelogs.field_name` = status.

### What you expected to happen

In the case of `issue_changelogs.field_name` = status, the value of `issue_changelogs.from_value` and `issue_changelogs.to_value` should only pick up one of the three values:
- TODO
- DONE
- IN_PROGRESS

### How to reproduce

- setup Devlake
- trigger the sub-task `convertChangelogs` of the Jira plugin
- check data of the table `issue_changelogs`

### Anything else

_No response_

### Version

main

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by mindlesscloud**
# Summary
fix #2691 [Bug][Jira] Incorrect values of issue_changelogs.from_value and issue_changelogs.to_value

### Does this close any open issues?
close #2691 

### Screenshots
Include any relevant screenshots here.

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

File: plugins/jira/tasks/issue_changelog_convertor.go [hunk 1]
```diff
@@ -127,8 +136,14 @@ func ConvertIssueChangelogs(taskCtx core.SubTaskContext) error {
 				}
 			}
 			if row.Field == "status" {
-				changelog.FromValue = getStdStatus(row.FromString)
-				changelog.ToValue = getStdStatus(row.ToString)
+				changelog.FromValue = ticket.IN_PROGRESS
+				changelog.ToValue = ticket.IN_PROGRESS
+				if fromStatus, ok := statusMap[row.FromString]; ok {
+					changelog.FromValue = fromStatus.StatusCategory
```
Comment (narrowizard) [inline plugins/jira/tasks/issue_changelog_convertor.go:142]: `getStdStatus(fromStatus.StatusCategory)`

File: plugins/jira/tasks/issue_changelog_convertor.go [hunk 2]
```diff
@@ -127,8 +136,14 @@ func ConvertIssueChangelogs(taskCtx core.SubTaskContext) error {
...
+				}
+				if toStatus, ok := statusMap[row.ToString]; ok {
+					changelog.ToValue = toStatus.StatusCategory
```
Comment (narrowizard) [inline plugins/jira/tasks/issue_changelog_convertor.go:145]: `getStdStatus(toStatus.StatusCategory)`

File: plugins/jira/tasks/status_collector.go [hunk 3]
```diff
@@ -48,12 +48,14 @@ func CollectStatus(taskCtx core.SubTaskContext) error {
 			Table: RAW_STATUS_TABLE,
 		},
 		ApiClient:     data.ApiClient,
-		UrlTemplate:   "api/2/status",
+		UrlTemplate:   "api/2/statuses/search",
```
Comment (narrowizard) [inline plugins/jira/tasks/status_collector.go:51]: Api `api/2/statuses/search` is not existed in jira server version `v8.18.1#818001-sha1:765ecc5`

Comment (narrowizard) [reply plugins/jira/tasks/status_collector.go:51]: image

**Review by narrowizard — APPROVED** [review]
LGTM

**Review by klesh — APPROVED** [review]
LGTM

**Pull request merged**